vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -466,6 +490,68 @@ and
 
         setup (siteProvider.GetProjectSite()) |> ignore
 
+    member private __.ProvideProjectSiteProvider(workspace:Workspace, project:Project) =
+        let visualStudioWorkspace = workspace :?> VisualStudioWorkspace
+        let hier = visualStudioWorkspace.GetHierarchy(project.Id)
+        {new IProvideProjectSite with
+             member this.GetProjectSite() =
+                 let compileItems () = [| for document in project.Documents do yield document.FilePath |]
+                 let compilerFlags () = [| "" |]
+                 let caption () = project.Name
+                 let projFileName () = project.FilePath
+                 let taskProvider = None
+                 let taskReporter = None
+                 let targetFrameworkMoniker = ""
+                 let projectGuid () = project.Id.Id.ToString()
+                 let creationTime = System.DateTime.Now
+                 let assemblyReferences () = 
+                    [|
+                        for reference in project.ProjectReferences do 
+                            let p = workspace.CurrentSolution.GetProject(reference.ProjectId)
+                            yield (p.OutputFilePath)
```

Comment:
@pilchie either works since we need metadata from the built dll.  Final might be better if the FCS does magic resolution for dependencies not specified in the project.